CHANGELOG.md

```diff
@@ -35,7 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - amazon.py removed due to broken scraper and no maintainer
 - newegg.py removed due to broken scraper and no maintainer
 - Removed path patching in main module
-
```

Comment:
Please add this blank line back in after your entry

Reply:
Done